### PR TITLE
Updated adilio command

### PR DESCRIPTION
**Pull request by adilio**
Updated adilio bot command to use the other emote.

### CODE REVIEW

File: commands.go [hunk 1]
```diff
@@ -266,7 +266,7 @@ func adilioMessage(s *discordgo.Session, m *discordgo.MessageCreate) {
 	span.AddField("command", "AdilioLol")
 
 	if strings.Contains(strings.ToLower(m.Message.Content), "lol") {
-		sendResponse(ctx, s, m.ChannelID, "<:adilolol:769432234880925727> <:adilol:769263097772245032>")
+		sendResponse(ctx, s, m.ChannelID, "<:adilio:769432234880925727> <:adilol:769263097772245032>")
```
Comment (corbob) [inline commands.go]: You'll want to include the emote ID: 788986373038145537

Comment (corbob) [inline commands.go]:
```suggestion
		sendResponse(ctx, s, m.ChannelID, "<:adilio:788826086628261889> <:adilol:769263097772245032>")

```
Need to use the ID of the emote.

**Review by corbob — CHANGES_REQUESTED** [review]

**Comment by corbob** [comment]
If you take out line 27 it should solve the issue where you're blocked from sending other commands... I can't suggest it because you didn't change it.

**Comment by ChrisLGardner** [comment]
> 
> 
> If you take out line 27 it should solve the issue where you're blocked from sending other commands... I can't suggest it because you didn't change it.

I've got an idea of how to handle this sort of thing so I'll get in a PR to fix that later today. In the mean time I'll get your fix added and merge this PR

**Pull request merged**